workable-module-connector/0.1/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,21 @@
+:connector-name: Workable Module
+:lc-connector-name: workable-module
+
+= {connector-name} Connector 0.1.0-BETA
+
+Anypoint Connector for {connector-name} ({connector-name} Connector) enables Mule applications to access the {connector-name} API.
+
```

Comment:
add the beta disclaimer here, like how i did in line 8 of index here:

https://github.com/mulesoft/docs-connectors/pull/3053/commits/57714fb86fa8a4d194a9d57049686fe4b7f4e22b